lib/table-utils.js

```diff
@@ -72,6 +72,72 @@ module.exports.getTripsForTripId = function (id) {
   });
 };
 
+module.exports.getAllTrips = function () {
+  var summaryQuery = new azureStorage.TableQuery(); // return all trips, no filter
+    
+
+    
+  // use promise instead
+  var tempFunc = tableService.queryEntities;
+  tableService.queryEntities = RSVP.denodeify(tableService.queryEntities);
+  
+  return tableService.queryEntities(nconf.get('trip_summaries_table_name'), summaryQuery, null)
+  .then( function(result) {
+
+  	var promises = [];
+    
+    result.entries.forEach(function (entity) {
+      var tripQuery = new azureStorage.TableQuery()
+        .where('PartitionKey eq ?', entity.PartitionKey._)
```

Comment:
due to changes from the storage data - we'll need to change the to
`'ParitionKey eq ?', entity.PartitionKey._ + entity.RowKey._`